apps/system/js/audio_channel_controller.js

```diff
@@ -9,11 +9,39 @@
   /**
    * AudioChannelController controls the audio channel to
    * play or pause.
+   *
+   * There are two ways to create an AudioChannelController instance.
+   * One is for System app, and the other one is for other apps. 
+   *
+   * Doing `new AudioChannelController('notification')`
+   * to create a controller for System app's notification audio channel.
+   *
+   * Doing `new AudioChannelController(musicApp, contentChannel)`
+   * to create a controller for Music app's content audio channel.
+   * 
+   * @param {AppWindow|String} app
+   * If you want to create a controller for System app's audio channel,
+   * use a String.
+   * If you want to create a controller for other app's audio channel,
+   * use a AppWindow.
+   * @param {BrowserElementAudioChannel} channel
+   * An audio channel belongs to the app.
    */
   var AudioChannelController = function(app, channel) {
-    this.app = app;
-    this.name = channel.name;
-    this._channel = channel;
+    if (typeof app === 'string') {
+      // The audio channel belongs to System app.
+      // Set a fake window ID for System app.
+      // It is used for generating ID of this AudioChannelController.
+      this.app = { instanceID: 'systemAppID' };
+      this.name = app;
```

Comment:
Weird statement...please reorder the arguments or new AudioChannelController with SystemWindow ref in SystemWindow